packages/ceramic-http-client/src/ceramic-http-client.ts

```diff
@@ -38,10 +37,10 @@ class CeramicClient implements CeramicApi {
         const result = await fetchJson(url)
         const { pinnedDocIds } = result
         return {
-          [Symbol.asyncIterator]() {
+          [Symbol.asyncIterator](): any {
```

Comment:
`AsyncIterable<string>` no?

Reply:
it should actually be `AsyncIterator<string, any, undefined>` if I'm not mistaken :)